zuul.d/nodeset.yaml

```diff
@@ -360,3 +360,20 @@
     nodes:
       - name: controller
         label: centos-9-stream-crc-2-36-0-xl
+
+- nodeset:
+    name: centos-9-medium-centos-9-crc-cloud-ocp-4-16-rc4-3xl
+    nodes:
+      - name: controller
+        label: cloud-centos-9-stream-tripleo-medium
+      - name: compute-0
+        label: cloud-centos-9-stream-tripleo
+      - name: crc
+        label: crc-cloud-ocp-4-16-0-rc4-3xl
```

Comment:
Usually, all I need is the nodeset. I'll upload a PR for STF with just that change.

In the meantime, I will need to check what the base-crc-cloud job does and how it differs from base extracted crc.